System.IO.Compression.ZipFile tests failed in Windows outer loop

See: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/outerloop_windows_nt_debug/131/consoleFull

There is not much output from the log.
````
02:17:11   xUnit.net console test runner (64-bit .NET Core)
02:17:11   Copyright (C) 2014 Outercurve Foundation.
02:17:11   
02:17:11   Discovering: System.IO.Compression.ZipFile.Tests
02:17:11   Discovered:  System.IO.Compression.ZipFile.Tests
02:17:11   System.IO.Compression.Tests -> d:\j\workspace\outerloop_win---069ccc6f\bin\Windows_NT.AnyCPU.Debug\System.IO.Compression.Tests\System.IO.Compression.Tests.dll
02:17:11   Starting:    System.IO.Compression.ZipFile.Tests
02:17:12   no future
02:17:12 d:\j\workspace\outerloop_win---069ccc6f\Tools\tests.targets(130,5): warning MSB3073: The command "d:\j\workspace\outerloop_win---069ccc6f\bin/tests/Windows_NT.AnyCPU.Debug/System.IO.Compression.ZipFile.Tests/dnxcore50/CoreRun.exe xunit.console.netcore.exe System.IO.Compression.ZipFile.Tests.dll  -xml testResults.xml -notrait category=nonwindowstests -notrait Benchmark=true  -notrait category=failing " exited with code 1. [d:\j\workspace\outerloop_win---069ccc6f\src\System.IO.Compression.ZipFile\tests\System.IO.Compression.ZipFile.Tests.csproj]
02:17:12 d:\j\workspace\outerloop_win---069ccc6f\Tools\tests.targets(137,5): error : One or more tests failed while running tests from 'System.IO.Compression.ZipFile.Tests' please check log for details! [d:\j\workspace\outerloop_win---069ccc6f\src\System.IO.Compression.ZipFile\tests\System.IO.Compression.ZipFile.Tests.csproj]
````

The only thing standing out is this `no future` message in the log. This message appears a couple of times in assertions in the [clrcompression source](https://github.com/dotnet/corefx/search?utf8=%E2%9C%93&q=%22no+future%22).

--- a/src/Native/Windows/CMakeLists.txt
+++ b/src/Native/Windows/CMakeLists.txt
@@ -28,7 +28,9 @@ endif ()
 
 string(TOUPPER ${CMAKE_BUILD_TYPE} UPPERCASE_CMAKE_BUILD_TYPE)
 if (UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG)
-    add_definitions(-DDEBUG)
+    # Do not define DEBUG. zlib has asserts under DEBUG for non-catastrophic cases,
+    # such as on bad user-provided inputs.  We leave NDEBUG defined, however,
+    # as other asserts should still be included.
 elseif (UPPERCASE_CMAKE_BUILD_TYPE STREQUAL RELEASE)
     add_definitions(-DNDEBUG)
 else ()

